ui/store/actions.js

```diff
@@ -1297,40 +1291,49 @@ export function addTokens(tokens) {
   };
 }
 
-export function removeSuggestedTokens() {
-  return (dispatch) => {
+export function rejectWatchAsset(suggestedAssetID) {
```

Comment:
[Q] is this name change strictly to align with the underlying provider method?

Reply:
Yes. And it also now rejects one pending/suggested asset. The change to handling a single asset at a time is not yet leveraged but I think it should be soon!